src/locale/locales/pt-BR/messages.po

```diff
@@ -1986,27 +1986,27 @@ msgstr "Descubra novos feeds"
 
 #: src/view/screens/Search/Explore.tsx:389
 msgid "Discover new feeds"
-msgstr ""
+msgstr "Descubra novos feeds"
 
 #: src/view/screens/Feeds.tsx:756
 msgid "Discover New Feeds"
 msgstr "Descubra Novos Feeds"
 
 #: src/view/com/profile/ProfileHeaderSuggestedFollows.tsx:108
 msgid "Dismiss"
-msgstr ""
+msgstr "Liberar"
 
 #: src/view/com/composer/Composer.tsx:612
 msgid "Dismiss error"
-msgstr ""
+msgstr "Liberar erro"
```

Comment:
maybe "Ocultar"? I know we are using that for "Hide" but feels better in this context

Reply:
Ocultar ok

Reply:
Ocultar erro OK!